monitoring/quickstart.rb

```diff
@@ -0,0 +1,34 @@
+require "google/cloud/monitoring"
+
+def quickstart
+  # [START monitoring_quickstart]
+  # Your Google Cloud Platform project ID
```

Comment:
Would it be possible to extract the project ID from credentials JSON file if it is available via environment variable?

Reply:
I suppose this should probably be done in separate PR. Please don't worry about it for now.

Reply:
ok